README.md

```diff
@@ -100,9 +100,11 @@ You can combine the command line options given below.
   ./candi.sh --prefix=Your/Prefix/Path
 ```
 
-#### Multiple build processes: ``[-j <N>]``, ``[--PROCS=<N>]``
+#### Multiple build processes: ``[-j<N>]``, ``[-j <N>]``, ``[--jobs=<N>]``
 ```bash
+  ./candi.sh -j<N>
```

Comment:
@tjhei I think we had once a discussion about that kind of option. Are you fine with having -jN and -j N?

Reply:
At least both works

Reply:
What was the final todo for this issue?

Reply:
@tjhei approved both forms and is fine with the change, so we keep both options here.